lightspeed/modules/proc_login-vscode-extension.adoc

```diff
@@ -5,7 +5,34 @@
 
 After installing and configuring the VS Code extension, you can log in to the Ansible Lightspeed service.
 
-.Procedure
+== Sign-in options
+
+{LightspeedShortName} provides different sign-in methods depending on whether you are using the cloud service or the on-premise deployment.
+
+* *Ansible Lightspeed on-premise deployments*
++
+Users are authenticated using your {PlatformName} login. 
++
+To sign in, you can use the *Connect* button or the *Accounts* button, and use the option *Sign in with Ansible Lightspeed to use Ansible* from VS Code. Once prompted, select *Log in with {PlatformNameShort}*, and log in with the authorization mechanism that your automation controller is configured with. 
+
+* *Ansible Lightspeed cloud service*
++
+Users are authenticated using Red Hat Single Sign-On (RH-SSO).
++
+To sign in, you can use the *Connect* button or the *Accounts* button and then select one of the following options:
+
+** *Sign in with Ansible Lightspeed to use Ansible*
+** *Sign in with Red Hat to use Ansible*
```

Comment:
Should this also use `{RHSSO} ` or is it explicitly using `Red Hat`?

Reply:
This is the UI label